src/components/MovieGrid/MovieGrid.js

```diff
@@ -1,40 +1,30 @@
-import React, { Component } from "react";
+import React from "react";
 
-import MovieItem from "../MovieItem/MovieItem";
+import MovieItem from "../MovieItem/";
 
 import "./MovieGrid.sass";
-import LoadingIndicator from "../../components/LoadingIndicator/LoadingIndicator";
+import LoadingIndicator from "../../components/LoadingIndicator";
 
-class MovieGrid extends Component {
-  shouldComponentUpdate(nextProps, nextState) {
-    if(nextProps.movies !== this.props.movies){
-      return true
-    } else if(nextProps.favs !== this.props.favs) {
-      return true
-    } else {
-      return false
-    }
-  }
+const MovieGrid = props => {
+  const { movies, handleFavButtonClicked, loadingIndicatorOff, favs } = props;
 
-  render() {
-    return (
-      <div className="movie-grid">
-        {this.props.movies
-          ? this.props.movies.map(({ imdbID, Title, Poster }) => (
-              <MovieItem
-                key={imdbID}
-                id={imdbID}
-                Title={Title}
-                Poster={Poster}
-                favs={this.props.favs}
-                handleFavButttonClicked={this.props.handleFavButttonClicked}
-              />
-            ))
-          : null}
-        {this.props.loadingIndicatorOff ? null : <LoadingIndicator />}
-      </div>
-    );
-  }
-}
+  return (
+    <div className="movie-grid">
+      {movies
```

Comment:
This check is quite unnecessary, because if you would default movies to an empty array nothing would be displayed.

Reply:
(So just add movies = [] to props deconstruction as a default value)